wbooks/src/app/interfaces/authorization.ts

```diff
@@ -1,14 +1,10 @@
 export interface AuthorizationState {
-  user: UserData | null;
+  user: User | null;
   userLoading: boolean;
   userError: UserError | null;
   userStatusCode: number;
 }
 
-export interface UserData {
-  data: User;
-}
-
 export interface UserError {
   errors: [string];
```

Comment:
Assuming you can get multiple errors. `string[]` is for an array of strings and `[string]` is for an array with a single string
```suggestion
  errors: string[];
```